release/src/Util.hs

```diff
@@ -81,6 +81,8 @@ data JarType
       -- ^ A java protobuf library (*-speed.jar).
     | Scala
       -- ^ A scala library jar, with source and scaladoc jars.
+    | JarJar
+      -- ^ A scala library jar, with source and scaladoc jars.
```

Comment:
Should this comment line be slightly different from the previous one? As it stands it makes it a bit difficult to understand why we need a separate type.

Reply:
absolutely that was dumb copypasta c83cea73522203e0b8e52776bff69422fabc054b